codeJE/config_tasks.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+# shellcheck disable=SC2034 # Ignore unused parameters.
+
+# Configuration of tasks for runtest.sh
+# (Cleans directory, modifies step activation, modifies JSON, generates step scripts.)
+
+# Mandatory functions:
+#   Clean                  Performs cleanup before (argument=1) and after (argument=2) running.
+#   AdjustJson             Modifies the JSON file.
+#   MakeScriptAli          Generates the AliPhysics script.
+#   MakeScriptO2           Generates the O2 script.
+#   MakeScriptPostprocess  Generates the postprocessing script.
+
+####################################################################################################
+
+# Steps
+DOCLEAN=1           # Delete created files (before and after running tasks).
+DOCONVERT=1         # Convert AliESDs.root to AO2D.root.
+DOALI=1             # Run AliPhysics tasks.
+DOO2=1              # Run O2 tasks.
+DOPOSTPROCESS=1     # Run output postprocessing. (Comparison plots. Requires DOALI=1 and/or DOO2=1)
+
+# Disable incompatible steps.
+[ "$ISINPUTO2" -eq 1 ] && { DOCONVERT=0; DOALI=0; }
+
+# O2 database
+DATABASE_O2="workflows.yml"
+MAKE_GRAPH=0        # Make topology graph.
+
+# Activation of O2 workflows
+# Trigger selection
+DOO2_TRIGSEL=1      # event-selection
+# QA
+DOO2_QA_EVTRK=0     # qa-event-track
+# User tasks
+DOO2_TASK_JETFINDER=1   # jet-finder
+DOO2_TASK_JETQA=1   # jetqa
+# Converters
+DOO2_MCCONV=0       # mc-converter
+DOO2_FDDCONV=0      # fdd-converter
+DOO2_COLLCONV=0     # collision-converter
+DOO2_ZDC=1          # zdc-converter
+DOO2_TRACKSELE=1    # trackselection, dca, propagation
+DOO2_TRACKEXTRA=1   # track-extra-converter
+DOO2_BC=1           # BC
+DOO2_DERIVED=1      # jet-deriveddata-producer
```

Comment:
- Provide correct workflow names in the comment.
- Group workflows properly. (`jet-deriveddata-producer` and others are not a converters.)
- Track selection and propagation are helper workflows that should be enabled automatically through the dependency chain, not explicitly with a switch.
- Fix the names to match the codeHF configuration.